README.md

```diff
@@ -35,6 +35,10 @@ Getting started
 5. Test: Visit http://localhost:8080/index.html
 6. (Optionally) Have your own [tanner](https://github.com/mushorg/tanner) service running.
 
+#### Docker build instructions
+1. `docker-compose build`
```

Comment:
Please make it explicit, that those commands should be run in snare folder. 

Also you can add a link for docker manual for those who are not familiar with a docker-compose
https://docs.docker.com/compose/gettingstarted/